var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -685,11 +685,8 @@ def configure_args(self):
         if 'fabrics=auto' not in spec:
             config_args.extend(self.with_or_without('fabrics'))
 
-        if spec.satisfies('@2.0.0:'):
-            if 'fabrics=xpmem' in spec and 'platform=cray' in spec:
-                config_args.append('--with-cray-xpmem')
-            else:
-                config_args.append('--without-cray-xpmem')
+        if spec.satisfies('@2.0.0: fabrics=xpmem platform=cray'):
+            config_args.append('--with-cray-xpmem')
```

Comment:
Should this uses the `with_or_without_xpmem` method from higher up in this file?